skorch/net.py

```diff
@@ -152,6 +154,31 @@ class NeuralNet:
       ``net.set_params(callbacks__print_log__keys_ignored=['epoch',
       'train_loss'])``).
 
+    predict_nonlinearity : callable, None, or 'auto' (default='auto')
```

Comment:
One minor concern: if one places a lambda function in `predict_nonlinearity`, it would not be pickle-able. Do you think its mentioning this in the docstring?